opentelemetry-sdk/CHANGELOG.md

```diff
@@ -10,18 +10,20 @@
   must now use `.with_resource(RESOURCE::default())` to configure Resource on
   logger provider.
 - Removed dependency on `ordered-float`.
+- Removed `XrayIdGenerator`, which was marked deprecated since 0.21.3. Use
```

Comment:
nit: can we add cargo.io link for aws create?

Reply:
added!